info.toml

```diff
@@ -701,4 +701,18 @@ path = "exercises/conversions/from_str.rs"
 mode = "test"
 hint = """
 If you've already solved try_from_into.rs, then this is almost a copy-paste.
-Otherwise, go ahead and solve try_from_into.rs first."""
\ No newline at end of file
+Otherwise, go ahead and solve try_from_into.rs first."""
+
+[[exercises]]
```

Comment:
Are we sure we want to put these at the end, e.g. behind arc and other, in my opinion more complicated stuff? 

I would maybe put them before `result` since Result kinda builds on what Option provides.

Reply:
Oh lol, I didn't know placement in here actually  mattered. And yeah, I think I should have put them before result.

Reply:
Fixed